automated-testing/container-testing/azdo-container-dev-test-release.md

```diff
@@ -0,0 +1,286 @@
+# Building Containers with Azure DevOps using DevTest Pattern
+
+In this documents, we highlight learnings from applying the DevTest pattern to container development in Azure DevOps through pipelines.
```

Comment:
Can you shed some light on why did you use this vs something else? I think it will be beneficial to understand your reasoning and the advantages of using this approach.